### PR TITLE
Update {b,x}cancel to have a 0.5px transition in gradient so that the…

### PR DESCRIPTION
**Pull request by kevinbarabash**
Addresses https://github.com/Khan/KaTeX/issues/760.  I'm not sure why the lines don't look antialiased in the screenshots.  I tested in Chrome on Mac OS (retina display) and Chrome on Windows.

Chrome on Mac OS
screen shot 2017-07-08 at 7 27 48 pm

Chrome on Windows
screen shot 2017-07-08 at 7 27 35 pm

### CODE REVIEW

File: static/katex.less [hunk 1]
```diff
@@ -794,41 +794,41 @@
...
     .cancel {
         background:
-           linear-gradient(to top left,
-               rgba(0,0,0,0) 0%,
-               rgba(0,0,0,0) ~"calc(50% - 0.04em)",
-               rgba(0,0,0,1) ~"calc(50% - 0.04em)",
-               rgba(0,0,0,1) ~"calc(50% + 0.04em)",
-               rgba(0,0,0,0) ~"calc(50% + 0.04em)",
-               rgba(0,0,0,0) 100%);
+            linear-gradient(to top left,
```
Comment (kohler) [inline static/katex.less]: the extra spaces in this diff are slightly annoying but also don't matter. 🤷‍♂️

Comment (kevinbarabash) [inline static/katex.less]: I can fix that.

**Comment by sthoch** [comment]
As I mentioned on the issue (#760), I advise testing on other browsers. I had some problems with gradients involving halfpixels some time ago - could be fixed now, could be not. IIrc, Safari (maybe on iOS) made some troubles.

**Comment by ronkok** [comment]
The screenshots below are, clockwise from top-left, Chrome, Firefox, Opera, and Safari. All are from Windows, on a Microsoft Surface, except the Safari, which is from BrowserStack.
cancel

I don't trust the pixelation shown for Safari, because BrowserStack shows the same pixelation for Opera, and as you can see, that pixelation disappears when taken from my own screen.  Someone with native equipment should re-test Safari.

I didn't test IE or Edge because this CSS doesn't run on IE or Edge. The CSS contains feature-detection that directs IE or Edge to render an SVG instead. Earlier testing showed that those two browsers go berserk when asked to render this linear-gradient combined with a `calc()`.

**Comment by ronkok** [comment]
I just looked at this on iOS Safari on my iPod Touch. All the lines look fine to me.

**Comment by kevinbarabash** [comment]
@ronkok thanks for testing across all of those different browsers.

**Pull request merged**